project/versions.scala

```diff
@@ -0,0 +1,77 @@
+package org.scalamacros
+package build
+
+import scala.compat.Platform.EOL
+import scala.io._
+import org.scalamacros.os._
+
+trait Versions { self: ScalamacrosBuild =>
+
+  lazy val LanguageVersion = sys.env.getOrElse("SCALA_VERSION", Scala211)
+  lazy val LanguageVersions = List(Scala210, Scala211, Scala212, Scala213, Dotty)
+  lazy val Scala210 = readScalaVersionFromDroneYml("2.10.x")
+  lazy val Scala211 = readScalaVersionFromDroneYml("2.11.x")
+  lazy val Scala212 = readScalaVersionFromDroneYml("2.12.x")
+  lazy val Scala213 = readScalaVersionFromDroneYml("2.13.x")
```

Comment:
Given the amount of effort that it takes to crosscompile sources and tests even now, I'm thinking of only supporting 2.13 in the Scala 2.x series.